src/main/python/tensorframes/core.py

```diff
@@ -208,6 +237,8 @@ def map_rows(fetches, dframe, feed_dict=None):
     :param dframe: a Spark DataFrame
     :return: a Spark DataFrame
     """
+    if isinstance(dframe, pd.DataFrame):
```

Comment:
Can you update the documentation above to reflect that the input can be either pandas or spark dataframes?